README.md

```diff
@@ -87,6 +87,18 @@ $lib->setToken($token->access_token)
 ### Authenticated
 
 
+#### Single User Applications
+
+If you only ever plan on interacting with a single user account feel free to hard code your access tokens into your app. This token should follow the spec described under authenticated requests.
```

Comment:
Lets drop the second sentence, and just keep it as...

`If you only ever plan on interacting with a single user account, feel free to hard code (or preferably configure) your access tokens into your app.`